lms/static/sass/course/_info.scss

```diff
@@ -45,342 +45,3 @@ $notification-background: rgb(255, 255, 255) !default;
     }
   }
 }
```

Comment:
Could you give me a clue why we do not remove this part of the `_info.scss` file? Where is it used?

Reply:
removed